server/supernova/CMakeLists.txt

```diff
@@ -206,6 +206,7 @@ target_link_libraries(supernova libsupernova)
 
 if(WIN32)
     if(NOT MSVC)
+      SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -municode")
```

Comment:
using SET in this way is very old-style cmake. i think you want target_link_libraries instead.

also, this will add this flag for MSYS builds too, do you know if that's a problem? i am not familiar enough with these systems to know but just thought i would ask. if you don't know, don't worry about it. @sonoro1234 may know?

Reply:
I dont use MSYS just mingw-64 gcc compiler from windows command line!!
I guess @dyfer tested the flag already

Reply:
thanks! in that case don't worry about MSYS support @dyfer